streams/src/main/java/org/apache/kafka/streams/state/internals/OffsetCheckpoint.java

```diff
@@ -66,6 +66,10 @@ public OffsetCheckpoint(final File file) {
      * @throws IOException if any file operation fails with an IO exception
      */
     public void write(final Map<TopicPartition, Long> offsets) throws IOException {
+        // if there is no offsets, skip writing the file to save disk IOs
+        if (offsets.isEmpty())
```

Comment:
should we use java style here? i.e., `if(..) {}`?

Reply:
ack